packages/web-api-scan-runner/src/runner/runner.ts

```diff
@@ -107,10 +93,7 @@ export class Runner {
             this.logger.trackEvent('ScanTaskSucceeded');
             pageScanResult.run = this.createRunResult('completed');
             pageScanResult.scanResult = this.getScanStatus(axeScanResults);
-            pageScanResult.reports = [
-                await this.saveScanReport(axeScanResults, 'sarif'),
-                await this.saveScanReport(axeScanResults, 'html'),
-            ];
+            pageScanResult.reports = await Promise.all(this.generateAndSaveScanReports(axeScanResults));
```

Comment:
Better move Promise.all() inside to the call map() as it bit misleading that promise.all() should be used here